sdk/eventgrid/Microsoft.Azure.EventGrid/tests/Microsoft.Azure.EventGrid.Tests.csproj

```diff
@@ -1,17 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.test.reference.props'))" />
   <PropertyGroup>
-    <PackageId>Microsoft.Azure.EventGrid.Tests</PackageId>
-    <Version>1.0.0</Version>
+    <AssemblyTitle>Microsoft Azure EventGrid Tests</AssemblyTitle>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
     <Description>Microsoft.Azure.EventGrid.Tests Class Library</Description>
-    <Authors>Microsoft Corporation</Authors>
-    <AssemblyName>Microsoft.Azure.EventGrid.Tests</AssemblyName>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
-    <ProjectReference Include="..\..\management\Management.EventGrid\Microsoft.Azure.Management.EventGrid.csproj" />
-    <ProjectReference Include="..\Microsoft.Azure.EventGrid\Microsoft.Azure.EventGrid.csproj" />
+    <ProjectReference Include="$(RepoSrcPath)\EventGrid\management\Management.EventGrid\Microsoft.Azure.Management.EventGrid.csproj" />
```

Comment:
The issue you are hitting in CI is because we are trying to build a management library but we don't have the management tools setup. Instead of doing this project reference lets change this to a package reference:
```
<PackageReference Include="Microsoft.Azure.Management.EventGrid" Version="4.0.1-preview" />
```

With this approach you can also undo all your changes to Directory.Build.mgmt.targets file as they will no longer be needed.